index.html

```diff
@@ -0,0 +1,86 @@
+dents or broken door. Do not install if oven is
+damaged.</p>
+
+  <p>Cabinet: Remove any protective film found on the microwave oven cabinet surface. Do not remove the light brown Mica cover that is attached to the oven cavity to protect the magnetron. </p>
+
+  <section class="main-section" id="Clock_Setting">
+  <header>Clock Setting</header>
+  <p>You'll be able to cook the most delicious meals with this oven but first let's set the clock of your oven</p>
+
+  <ol>
+    <li> Press “Clock“ , “00:00” will display.</li>
+    <li> Press the number keys and enter the current time. For example, time is 10:12 now, please
+	 press “1,0,1,2” in turn </li>
+   <li> Press “Clock“ to finish clock setting. “:” will flash and the clock will be lighted.</li>
+   <li> If the numbers input are not within the range of 1:00--12:59, the setting will be invalid until
+valid numbers are input.</li>
+
+  <p>Now everything will be ready to cook your first meal!</p>
+
+  <section class="main-section" id="Operation">
+  <header>Operation</header>
+  <p> With this wonderful Microwave you'll be able to reheat your favorite Pizza and cook Potatos ONLY but the'll be the best.</p>
+  For Pizza Reheat follow this steps:
+
+  <ol>
+    <li>Press “PIZZA“ repeatedly until the number you wish appears in the display, “4.0”,”8.0”,”14.0” will display in order. For example, press “PIZZA“ once, “4.0” appears</li>
+    <li>Press “START/ADD 30sec“ to cook, buzzer sounds once. When cooking finish, buzzer sounds five times and screen will display the correct time. </li>
+
+<br>
+For Potato cooking:
+  <ol>
+  <li>Press “POTATO“ repeatedly until the number you wish appears in the display, “1”,”2”,”3" will display in order.</li>
+  <li>Press “START/ADD 30sec“ to cook, buzzer sounds once. When cooking finish, buzzer sounds five times and screen</li>
+
+  <p> That's all! Enjoy your pizza an Potatoes! </p>
+   
+  <section class="main-section" id="Maintenance">
+  <header>Maintenance</header>
+
+  <p>Make sure cleaning of the microwave is done regularly</p>
+  <p>Check your problem by using the chart in the Manual Appendix and try the solutions for each problem. If the
+microwave oven still does not work properly, contact the nearest authorized service center</p>
+
+  <section class="main-section" id="Warranty_information">
+  <header>Warranty information</header>
+  <p>This warranty applies to products purchased and used in the U.S and Canada</p>
+
+  <p>This warranty does not cover damages resulting from use not in conformity
+with the printed directions, accident, misuse or abuse, alternation, the affixing of any attachments not provided with the product, loss of parts, or subjecting the appliance to any but the specified voltage. (Read directions carefully.) This warranty is void if the product is used for other than single-family household use or subjected to any voltage and waveform other than as specified on the rating label (e.g., 120V ~ 60 Hz). 
+
+
+     </main> 
+
+
+
+
```

Comment:
Kindly, delete unnecesary blank lines, spaces, or indentations. For readability, add two spaces of indentation. Do not use the tab key. This will make your code look cleaner.